ts/defs.bzl

```diff
@@ -463,6 +463,19 @@ def ts_project(
         **kwargs
     )
 
+def _default_srcs(allow_js, resolve_json_module):
+    """Returns a list of srcs for ts_project when srcs is not provided."""
+    srcs = []
+    include = ["**/*.ts", "**/*.tsx"]
+    exclude = []
```

Comment:
Just set exclude to final value below instead?

Reply:
If `glob(exclude)` accepted `None` then I would. But I'm mainly just moving this code and don't want to think about it anymore 😅

(I did the other suggestion re the useless `src = []` though)

Reply:
Oh I see it's only set inside the condition. Hard to tell on phone :) you could also override the car instead of extend, that would save some keystrokes :)